point/chain.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "point",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Point",
+  "chain_id": "point_10687-1",
+  "bech32_prefix": "point",
```

Comment:
Still failing checks. Please register this bech32 prefix to SLIP173:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done. Please check.
https://github.com/satoshilabs/slips/pull/1404

Reply:
Thanks!